snot.py

```diff
@@ -245,18 +232,27 @@ def options(self, parser, env=os.environ):
 
     def configure(self, options, conf):
         super(SlickAsSnotPlugin, self).configure(options, conf)
-        global config, testrun
         assert isinstance(conf, nose.config.Config)
+        self.options = options
```

Comment:
Store options in class so they can be used in `addSlickTestrun` on line 239. :point_down: